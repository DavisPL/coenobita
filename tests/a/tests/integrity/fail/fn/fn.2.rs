#![allow(unused_variables)]
#![allow(unused_assignments)]

#[cnbt::tag({*}{*} fn({a,bin}{a,bin}) -> {a,bin}{a,bin})]
fn id(x: i32) -> i32 {
    x
}

fn main() {
    #[cnbt::tag({bin}{bin})]
    let x = id(5);
}