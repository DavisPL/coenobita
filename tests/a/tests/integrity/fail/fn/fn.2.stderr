error: expected {bin}{bin}, found {a,bin}{a,bin}
  --> tests/integrity/fail/fn/fn.2.rs:11:13
   |
11 |     let x = id(5);
   |             ^^^^^
